js/modules/k6/grpc/client.go

```diff
@@ -212,15 +205,46 @@ func (t transportCreds) ClientHandshake(ctx context.Context,
 	return out, auth, err
 }
 
+func (c *Client) dial(
+	ctx context.Context,
+	addr string,
+	reflect bool,
+	options ...grpc.DialOption,
+) error {
+	opts := []grpc.DialOption{
+		grpc.WithBlock(),
+		grpc.FailOnNonTempDialError(true),
+		grpc.WithStatsHandler(c),
+		grpc.WithReturnConnectionError(),
+	}
+	opts = append(opts, options...)
+
+	var err error
+	c.conn, err = grpc.DialContext(ctx, addr, opts...)
+	if err != nil {
+		return err
+	}
+
+	if !reflect {
+		return nil
+	}
+
+	err = c.reflect(ctx)
+	if err != nil {
+		return err
+	}
+
+	return err
```

Comment:
```suggestion
	return c.reflect(ctx)
```
Maybe also possible to rename a `reflect` method to `extractError` or something more human-friendly :thinking:

Reply:
Why do you want rename reflect to extractError? `reflect` is related to the grpc reflection feature in this case

Reply:
Ah, true, missed that.

In general, wanted to rename since reflect too general, but we can keep it as is